recipes/rye/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "0.0.32" %}
```

```suggestion
{% set version = "0.32.0" %}
```

ref: https://github.com/astral-sh/rye/releases/tag/0.32.0